Lombiq.DataTables.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -83,7 +120,10 @@ public static async Task TestDataTableIndexBasedProviderAsync(this UITestContext
         await context.TestDataTableProviderAsync();
     }
 
-    public static async Task TestDataTableProviderAsync(this UITestContext context)
+    public static Task TestDataTableProviderAsync(this UITestContext context) =>
+        context.TestDataTableProviderAsync(testExport: true);
```

Comment:
Why not just `bool testExport = true` in the other overload?

Reply:
Because the compatibility checking is dumb:
>  Error: /usr/share/dotnet/sdk/8.0.301/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.ApiCompat.ValidatePackage.targets(39,5): error CP0002: Member 'System.Threading.Tasks.Task Lombiq.DataTables.Tests.UI.Extensions.TestCaseUITestContextExtensions.TestDataTableProviderAsync(this Lombiq.Tests.UI.Services.UITestContext)' exists on [Baseline] lib/net8.0/Lombiq.DataTables.Tests.UI.dll but not on lib/net8.0/Lombiq.DataTables.Tests.UI.dll [/home/runner/work/Orchard-Data-Tables/Orchard-Data-Tables/Lombiq.DataTables.Tests.UI/Lombiq.DataTables.Tests.UI.csproj]

Reply:
Ah. It's not dumb, but it's about binary compatibility. It's not the same method signature anymore, so it's a breaking change, even if C# consumers wouldn't notice it. You can suppress it, because we don't care about languages not supporting default parameters.